pom.xml

```diff
@@ -217,6 +231,11 @@
         <version>2.6</version>
       </dependency>
       <dependency>
+        <groupId>commons-io</groupId>
+        <artifactId>commons-io</artifactId>
+        <version>2.5</version>
```

Comment:
@benzonico `commons-io` is used just in few places, where most likely it can be replaced by Java 8 APIs or Guava.